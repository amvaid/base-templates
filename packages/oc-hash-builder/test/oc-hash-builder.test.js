const hashBuilder = require('../index.js');

test('Correctly generate hash out of strings', () => {
  const string = `
    const myName = 'Nick';
    const say = something => console.log(something);
    say(myName);
  `;
  expect(hashBuilder.fromString(string)).toMatchSnapshot();
});
