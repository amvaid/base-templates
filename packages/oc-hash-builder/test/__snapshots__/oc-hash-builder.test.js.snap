// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly generate hash out of strings 1`] = `"df2ab5a5885859354404f704107a8a80ef087d68"`;
